providers/vsphere/go.mod

```diff
@@ -0,0 +1,62 @@
+module go.mondoo.com/cnquery/providers/vsphere
+
+replace go.mondoo.com/cnquery => ../..
```

Comment:
Is this intended?

Reply:
yes, this allows us to always use the latest cnquery version